node_filesystem/templates/bird.cfg.template

```diff
@@ -63,20 +75,40 @@ template bgp bgp_template {
   graceful restart;  # See comment in kernel section about graceful restart.
 }
 
-{{if ls "/config/bgp_peer_rr_v4"}}
-# Peering with configured neighbors.
-{{range gets "/config/bgp_peer_rr_v4/*"}}{{$nums := split .Value "."}}{{$id := join $nums "_"}}
-# For peer {{.Value}}
+{{/* ------------- Node-to-node mesh ------------- */}}
+{{if lsdir "/host"}}
```

Comment:
Is this necessary?  Don't we have to at least have ourselves in the `/host` directory?